### PR TITLE
feat: multi-env sample for spfx

### PR DESCRIPTION
**Pull request by nliu-ms**

### CODE REVIEW

File: todo-list-SPFx/.fx/configs/localSettings.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (qinezh) [inline todo-list-SPFx/.fx/configs/localSettings.json:1]: localSettings.json should be added to .gitignore. It's a bug from upgrade side and is going to be fixed in this PR: https://github.com/OfficeDev/TeamsFx/pull/2940

Comment (qinezh) [reply todo-list-SPFx/.fx/configs/localSettings.json:1]: @nliu-ms The file need to be removed?

File: todo-list-SPFx/README.md [hunk 2]
```diff
@@ -67,11 +67,11 @@
 ![appdisplay](images/appdisplay.png)
 
 ## (Optional) Debug
-Debug the app with SharePoint WorkBench in VSCode.
-1. Navigate to [launch.json](.vscode/launch.json), replace `enter-your-SharePoint-site` with your SharePoint site, eg. `https://{your-tenant-name}.sharepoint.com/sites/{your-team-name}/_layouts/workbench.aspx`.
-1. In Debug mode, select "Hosted workbench"(SharePoint workbench) or "Teams workbench" and press start button. The browser will be opened and you may need to sign in with your M365 account.
-1. Click the plus button in the middle, and select `TodoList`, the webpart will show up in the workbench.
- ![HostedWokbench](images/Workbench.png)
+Debug the app with Teams workbench in VSCode.
+
```
Comment (zyxiaoyuer) [inline todo-list-SPFx/README.md:71]: "Debug the app with Teams workbench" is good. 
".. Teams workbench in VS Code" is a bit confused.

File: todo-list-SPFx/README.md [hunk 3]
```diff
@@ -67,11 +67,11 @@
...
+1. In Debug mode, select `Teams workbench (Edge)` or `Teams workbench (Chrome)` and press start button. The browser will be opened and you need to sign in Teams with your M365 account.
+2. Click `Add to a team`.
+3. You should see the app running in your Teams.
```
Comment (zyxiaoyuer) [inline todo-list-SPFx/README.md:75]: we should have a screenshot of app running in Teams context.

File: todo-list-SPFx/README.md [hunk 4]
```diff
@@ -67,11 +67,11 @@
...
 
 ## (Optional) Debug
-Debug the app with SharePoint WorkBench in VSCode.
-1. Navigate to [launch.json](.vscode/launch.json), replace `enter-your-SharePoint-site` with your SharePoint site, eg. `https://{your-tenant-name}.sharepoint.com/sites/{your-team-name}/_layouts/workbench.aspx`.
-1. In Debug mode, select "Hosted workbench"(SharePoint workbench) or "Teams workbench" and press start button. The browser will be opened and you may need to sign in with your M365 account.
-1. Click the plus button in the middle, and select `TodoList`, the webpart will show up in the workbench.
- ![HostedWokbench](images/Workbench.png)
+Debug the app with Teams workbench in VSCode.
+
+1. In Debug mode, select `Teams workbench (Edge)` or `Teams workbench (Chrome)` and press start button. The browser will be opened and you need to sign in Teams with your M365 account.
```
Comment (zyxiaoyuer) [inline todo-list-SPFx/README.md:72]: should have a screenshot here

**Review by qinezh — APPROVED** [review]

**Pull request merged**